### ISSUE TITLE
Any plans for a 1.8 version soon?

### ISSUE THREAD

**Issue by ProfessorProspector** [SonarSonic/Practical-Logistics#36]
Just wondering...looks like a cool mod, and it might be cool to add some compat with TechReborn (which only has active development on 1.8).

**Comment by SonarSonic** [comment]
Yeah I will definitely update eventually, I'm not sure when yet though.

I'm really confused by licensing, which type should I be using?

**Comment by ProfessorProspector** [comment]
It boils down to what you want people to do with your software. Here is a helpful page: choosealicense.com/

**Comment by SonarSonic** [comment]
Thanks, I've added the license now :)
